src/MySql/docs/Update-AzMySqlServer.md

```diff
@@ -39,27 +39,23 @@ Use Update-AzMySqlConfiguration instead if you want update server parameters suc
 
 ## EXAMPLES
 
-### Example 1: Update MySql server by resource group and server name
```

Comment:
You should commit the change of  examples/Update-AzMySqlServer.md

Reply:
> > > Can you provide the autorest cmd you use to generate the code?
> > 
> > 
> > It was autorest.cmd --use:@autorest/powershell@3.0.403 <- this command from the email you sent me last time..! Do I need to try with different command?
> 
> Which powershell version are you using?

I'm using pwsh 7.0.3 version,,

Reply:
Same as above..